src/components/data-chart/index.tsx

```diff
@@ -0,0 +1,471 @@
+import {
+  Button,
+  CircularProgress,
+  Dialog,
+  Divider,
+  Icon,
+  Scrim,
+  Typography,
+} from '@equinor/eds-core-react';
+import { clear } from '@equinor/eds-icons';
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import { Chart } from 'react-google-charts';
+import { getJson } from '../../dynatrace-api/api-helpers';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import { configVariables } from '../../utils/config';
+import './style.css';
+
+const AvailabilityCharts = () => {
+  const [error, setError] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [availabilityItems, setAvailabilityItems] = useState([]);
+  const [statusCodeItems, setStatusCodeItems] = useState([]);
+  const [visibleScrim, setVisibleScrim] = useState(false);
+
+  function timeDuration(date) {
```

Comment:
remember to type functions and variables

for example:
```suggestion
  function timeDuration(date: Date): string {
```